build.gradle.kts

```diff
@@ -245,3 +246,9 @@ subprojects {
 tasks.wrapper {
   distributionType = Wrapper.DistributionType.ALL
 }
+
+// Fix until 1.6.20 https://youtrack.jetbrains.com/issue/KT-49109
+rootProject.plugins.withType(NodeJsRootPlugin::class.java) {
+  rootProject.the<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension>().nodeVersion =
+    "16.13.0"
+}
```

Comment:
Kotlin 1.6.20 will fix this.

[KT-49530](https://youtrack.jetbrains.com/issue/KT-49530)

Reply:
Yep, in the comment above it.